cpp/compiled_grammar_impl.h

```diff
@@ -51,14 +51,15 @@ struct AdaptiveTokenMask {
   };
   StoreType store_type;
 
-  static constexpr int USE_BITSET_THRESHOLD = 200;
+  static constexpr int USE_BITSET_THRESHOLD = 1000;
```

Comment:
Side note: 1000 is a better threshold that balances the size of bitset storage / indice storage.